Assets/Plugins/CountlySDK/Models/CountlyEventModel.cs

```diff
@@ -64,7 +63,8 @@ public CountlyEventModel()
 
         [JsonProperty("dow")] public int DayOfWeek { get; set; }
 
-        [JsonProperty("tz")] public double Timezone { get; set; }
+        [JsonIgnore]
+        public double Timezone { get; set; }
```

Comment:
this should be made private and a depracation tag added

Reply:
well, after making it private do we need to make it deprecate?

Reply:
Yes. I didn't explicitly mention that for no reason.

Reply:
well, ok.

Reply:
any message on deprecation?

Reply:
**CountlyEventModel** is public we should make it internal.

Reply:
"any message on deprecation?" 
just that this field sholdn't be used as it will be removed.

"CountlyEventModel is public we should make it internal."
sure, go ahead